tests/services/plugins/plugin_folder/sub_folder/helpers.py

```diff
@@ -0,0 +1,2 @@
+def custom_function():
+    pass
```

Comment:
Maybe it would be good to also call this function in the `main.py` to test sub-folders in the e2e test (that the build doesn't fail)

Reply:
yes good point